### ISSUE TITLE
Consistency in use of "state flag" vs. "civil flag"

### ISSUE THREAD

**Issue by aplaice** [anki-geo/ultimate-geography#111]
(Issue discovered while trying to replace PNG flags with SVG versions.)

Since there is little agreement even among reliable sources (see below), this is rather low priority, but I'm opening this in case a vexillologist eventually turns up here.

The "[state flag](https://en.wikipedia.org/wiki/State_flag)" and "[civil flag](https://en.wikipedia.org/wiki/Civil_flag)" generally differ by the inclusion of a coat of arms in the "state flag", but not the "civil flag". The "state flag" is used by the government, while the "civil flag" can be used by anyone. (See also: [national flag](https://en.wikipedia.org/wiki/National_flag#On_land).)

Currently, for Bolivia we [use](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-bolivia.png) the [state flag](https://commons.wikimedia.org/wiki/File:Flag_of_Bolivia_(state).svg) and not the [civil flag](https://commons.wikimedia.org/wiki/File:Flag_of_Bolivia.svg) (as [described on Wikipedia](https://en.wikipedia.org/wiki/Flag_of_Bolivia)).

In contrast, for Peru we [use](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-peru.svg) the [civil flag](https://en.wikipedia.org/wiki/File:Flag_of_Peru.svg) and not the [state flag](https://en.wikipedia.org/wiki/File:Flag_of_Peru_(state).svg) (as [described on Wikipedia](https://en.wikipedia.org/wiki/Flag_of_Peru)). Similarly, for Austria, we [use](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-austria.svg) the [civil](https://en.wikipedia.org/wiki/File:Flag_of_Austria.svg) and not the [state](https://en.wikipedia.org/wiki/File:Flag_of_Austria_(state).svg) flag (see [description on Wikipedia](https://en.wikipedia.org/wiki/Flag_of_Austria)). 

Germany, among others, also has a ["state flag"](https://commons.wikimedia.org/wiki/File:Flag_of_Germany_(state).svg), but for some reason no sources use it as "*the*" flag of Germany (whatever one means by "*the*" flag).

On the other hand, almost everybody presents the [state flag](https://en.wikipedia.org/wiki/Flag_of_Spain) and not the "civil flag" as the flag of Spain, such that even on Wikimedia Commons, the state flag is listed as "the [flag of Spain](https://en.wikipedia.org/wiki/File:Flag_of_Spain.svg)", while the civil flag is the [flag of Spain (civil)](https://en.wikipedia.org/wiki/File:Flag_of_Spain_(Civil).svg).

I've looked at some hopefully reliable sources and summarised the info in a table below:

| Country | Our current use | Placed first by Wikipedia | Encyclopaedia Britannica | CIA factbook | My 90s paper Encyclopaedia | DK's "Flag" |
|---------|-----------------|---------------------------|--------------------------|--------------|----------------------------|-------------|
| Bolivia | [State](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-bolivia.png) | [State](https://en.wikipedia.org/wiki/Flag_of_Bolivia) | [State](https://www.britannica.com/topic/flag-of-Bolivia) | [State](https://www.cia.gov/library/publications/resources/the-world-factbook/geos/bl.html) | Civil | Civil |
| Peru    | [Civil](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-peru.svg) | [State](https://en.wikipedia.org/wiki/Flag_of_Peru) | [State](https://www.britannica.com/topic/flag-of-Peru) | [State](https://www.cia.gov/library/publications/resources/the-world-factbook/geos/pe.html) | Civil | Civil |
| Austria | [Civil](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-austria.svg) | [Civil](https://en.wikipedia.org/wiki/Flag_of_Austria) | [State](https://www.britannica.com/topic/flag-of-Austria) | [Civil](https://www.cia.gov/library/publications/resources/the-world-factbook/geos/au.html) | Civil | Civil |
| Germany | [Civil](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-germany.svg) | [Civil](https://en.wikipedia.org/wiki/Flag_of_Germany) | [Civil](https://www.britannica.com/topic/flag-of-Germany) | [Civil](https://www.cia.gov/library/publications/resources/the-world-factbook/geos/gm.html) | Civil | Civil |
| Spain   | [State](https://github.com/axelboc/anki-ultimate-geography/blob/master/src/media/ug-flag-spain.png) | [State](https://en.wikipedia.org/wiki/Flag_of_Spain) | [State](https://www.britannica.com/topic/flag-of-Spain) | [State](https://www.cia.gov/library/publications/resources/the-world-factbook/geos/sp.html) | Civil | Civil |

## Recommendations (?)

If I were to make any recommendations, which I'm not really qualified to do, it would be to follow Wikipedia's lead, and use whichever flag they list first (i.e. replace our Peru flag with the state flag; I'm not sure if any other flags would "need" to be changed under this policy). However, not messing around with the flags when there aren't any great criteria for the change might be a better option...

At least I've learned something more about world flags, which is, after all, part of the point of this deck. :)

**Comment by axelboc** [comment]
Very interesting, indeed! Totally up for following Wikipedia's lead as we've done so far in this deck. Let's go and replace the flag of Peru, then! I've gone through all the flags and found a few others:

- [Belgium](https://en.wikipedia.org/wiki/Flag_of_Belgium) (narrower)
- [Costa Rica](https://en.wikipedia.org/wiki/Flag_of_Costa_Rica) (with emblem)
- [Venezuela](https://en.wikipedia.org/wiki/Flag_of_Venezuela) (with emblem)

**Comment by aplaice** [comment]
Wow! Thanks for going through all the flags. In that case, these four flags should indeed be replaced.

Do you want to do it or should I open a pull request?

**Comment by axelboc** [comment]
Since you're offering... 😁
Thanks a lot for helping out so much.

**Comment by ollie299792458** [comment]
Wikipedia has since changed their mind on this (listing the "plain" civil flag first), should this be changed back? (I was surprised by the defaced Peruvian flag just now, was expecting the "plain" one)

Happy to make a new issue instead?

**Comment by ollie299792458** [comment]
Same issue with Venezuela, top flag is now civil flag again: https://en.m.wikipedia.org/wiki/Flag_of_Venezuela

In general, I think the civil flag is the one people most associate with a country, currently the deck appears "wrong" to people, as the Peru & Venezuela civil flags are the "correct" flags.

My suggestion is, rather than going on Wikipedia order (which clearly varies), we should go with  the civil flag unless the article clearly says the state flag is the preferred one (I don't personally know of any cases where this is the case).

**Comment by aplaice** [comment]
I'm slightly conflicted, in the case of Peru and Venezuela (mainly because I'd prefer avoiding switching between the two versions, as it must be extremely annoying for our users) and very conflicted in the general case.

It does seem that in both cases the flag order has stabilised both for Peru and Venezuela.

For Venezuela the change was AFAICT [made in December 2020](https://en.wikipedia.org/w/index.php?title=Flag_of_Venezuela&oldid=994443473).

For Peru, it was [made in January 2020](https://en.wikipedia.org/w/index.php?title=Flag_of_Peru&oldid=938336667) followed by a pretty protracted edit war.

However, Britannica, for example, uses the state flag in its articles for both [Peru](https://www.britannica.com/place/Peru) and [Venezuela](https://www.britannica.com/place/Venezuela).

(As a comparison point, of people who seem to have also given the question of country flag design, throughout the world, some explicit thought) the [`country-flags`](https://github.com/hampusborgos/country-flags/) seems to use the state flag for [Peru](https://github.com/hampusborgos/country-flags/blob/main/svg/pe.svg), but the civil flag for [Venezuela](https://github.com/hampusborgos/country-flags/blob/main/svg/ve.svg).  However, in neither of these two cases have they provided detailed [sources](https://github.com/hampusborgos/country-flags/blob/main/scripts/flags.json) (unlike the way they've provided explicit sources for other countries).

Hence, for Peru and Venezuela, I think that we should, indeed, switch to the civil flags, as you write.

<hr/>


However, I'm not convinced about the general case.

In the case of [Belgium](https://en.wikipedia.org/wiki/Flag_of_Belgium) I'd continue using the national flag rather than the civil ensign.  For Costa Rica, I'm not sure, but the order on Wikipedia [seems not to have changed](https://en.wikipedia.org/wiki/Flag_of_Costa_Rica) (i.e. state flag first).  (For Spain, yes, obviously, we should use the national flag (with the coat of arms), like you state.)

**Comment by ollie299792458** [comment]
@aplaice that all makes sense.

And if we hit examples that match the Wikipedia rule, but don't "seem" right, maybe the rule needs revisiting, but I can't find any more for now.

**Comment by josealberto4444** [comment]
Sorry for replying after 2 months, but for me these changes are quite controversial. For Peru, I even asked a friend from there and, although she agrees that the civil flag is more used than the state flag, she says it is only due to its simplicity. For her, studying that the Peru flag is the civil Peru flag is wrong, the flag is incomplete (I assume that they study the state flag as the Peru flag in school).

She even told me that it's the same case in Spain, where, when we paint flags or do handmade flags, they are the civil version (as the state version is too difficult to make), but that does not seem to imply that the civil version is the most used, and everyone seems to agree on that.

Summing up, my point is not that the flag of Peru should remain as the state version, but simply that the topic is very controversial and we should be very cautious about our criteria. Maybe even more than Wikipedia. For example, as a proposal, we may agree on a set of 3 or 5 sources (Wikipedia, Britannica and some more — maybe Larousse) to check and use the most repeated version in all of them.

**Comment by aplaice** [comment]
Thanks very much for the detailed background on Peru and Spain!

On the object level, that strongly suggests that Wikipedia should display the state flag.

> For example, as a proposal, we may agree on a set of 3 or 5 sources (Wikipedia, Britannica and some more — maybe Larousse) to check and use the most repeated version in all of them.

Interesting.  As always, I'm hesitant against using a larger set of sources, but maybe in this case, it might be worthwhile.

One might argue that this is a situation where Wikipedia is particularly unreliable because there are two almost equally good choices.  However, this doesn't differ much from the case with, say, multiple possibilities for the capital(s).

There's also the question of choice of sources — we don't want anything that's likely just (directly or indirectly) mirroring Wikipedia.

Larousse is probably an acceptable third source.  Treccani might have been OK, if it didn't apparently [omit flags](https://www.treccani.it/enciclopedia/peru/).  Norway's SNL might be another.

[Here's](https://www.europarl.europa.eu/RegData/etudes/IDAN/2018/630347/EPRS_IDA(2018)630347_EN.pdf) an interesting list of European "national" encyclopedias, that might be "independent" of Wikipedia (obviously not that we should constrain ourselves to European sources...).

The [Great Russian Encyclopedia](https://en.wikipedia.org/wiki/Great_Russian_Encyclopedia) and [Baidu Baike](https://en.wikipedia.org/wiki/Baidu_Baike) are some other possibilities.  (I don't think that their bias is likely to be relevant, in the case of flag choice...)

**Comment by aplaice** [comment]
In the interest of going forward, here's an overview of what the various Encyclopedia's say on the topic of Bolivia, Peru and Spain:















| Flag    | Wikipedia (flag page)                                                                                                                                                                                                       | Wikipedia (country page)                                                                                                                                                                         | Britannica                                                                                                                                   | Larousse                                                                                                                                                          | SNL                                                                                          | Baidu Baike                                                                                                                                                                                               | Great Russian Encyclopedia                          |
|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
| Bolivia | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Flag_of_Bolivia.svg/374px-Flag_of_Bolivia.svg.png) ([civil](https://en.wikipedia.org/wiki/Flag_of_Bolivia))                                                   | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b3/Bandera_de_Bolivia_%28Estado%29.svg/188px-Bandera_de_Bolivia_%28Estado%29.svg.png) ([state](https://en.wikipedia.org/wiki/Bolivia)) | ![](https://cdn.britannica.com/54/6754-050-897FBD93/Flag-Bolivia.jpg?w=400&h=235&c=crop) ([state](https://www.britannica.com/place/Bolivia)) | ![](https://www.larousse.fr/encyclopedie/data/vignettes/1009460-Drapeau_de_la_Bolivie.jpg) ([civil](https://www.larousse.fr/encyclopedie/pays/Bolivie/109362))    | ![](https://media.snl.no/media/1471/standard_f-bolivi.gif) ([civil](https://snl.no/Bolivia)) | ![](https://bkimg.cdn.bcebos.com/pic/b2de9c82d158ccbf6c81cf695894ab3eb13533faeb06?x-bce-process=image/resize,m_lfit,w_536,limit_1) ([state](https://baike.baidu.com/item/玻利维亚/61176))                 | ([civil](https://bigenc.ru/geography/text/4224650)) |
| Peru    | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/Flag_of_Peru.svg/383px-Flag_of_Peru.svg.png) ([civil](https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/Flag_of_Peru.svg/383px-Flag_of_Peru.svg.png)) | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/Flag_of_Peru.svg/188px-Flag_of_Peru.svg.png) ([civil](https://en.wikipedia.org/wiki/Peru))                                         | ![](https://cdn.britannica.com/48/3448-050-1EFC8CF3/Flag-Peru.jpg?w=400&h=235&c=crop) ([state](https://www.britannica.com/place/Peru))       | ![](https://www.larousse.fr/encyclopedie/data/vignettes/1009572-Drapeau_du_P%C3%A9rou.jpg) ([state](https://www.larousse.fr/encyclopedie/pays/P%C3%A9rou/137673)) | ![](https://media.snl.no/media/8877/standard_f-peru.gif) ([civil](https://snl.no/Peru))      | ![](https://bkimg.cdn.bcebos.com/pic/30adcbef76094b36fdbf346ea9cc7cd98d109d07?x-bce-process=image/resize,m_lfit,w_536,limit_1) ([state](https://baike.baidu.com/item/%E7%A7%98%E9%B2%81/258354))          | ([civil](https://bigenc.ru/geography/text/2775845)) |
| Spain   | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/Bandera_de_Espa%C3%B1a.svg/300px-Bandera_de_Espa%C3%B1a.svg.png) ([state](https://en.wikipedia.org/wiki/Flag_of_Spain))                                       | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/Bandera_de_Espa%C3%B1a.svg/188px-Bandera_de_Espa%C3%B1a.svg.png) ([state](https://en.wikipedia.org/wiki/Spain))                    | ![](https://cdn.britannica.com/36/4336-050-056AC114/Flag-Spain.jpg?w=400&h=235&c=crop) ([state](https://www.britannica.com/place/Spain))     | ![](https://www.larousse.fr/encyclopedie/data/vignettes/1009665-Drapeau_de_l_Espagne.jpg) ([state](https://www.larousse.fr/encyclopedie/pays/Espagne/118438))     | ![](https://media.snl.no/media/10422/standard_f-spania.gif) ([state](https://snl.no/Spania)) | ![](https://bkimg.cdn.bcebos.com/pic/c2cec3fdfc039245a48d86438e94a4c27d1e2545?x-bce-process=image/resize,m_lfit,w_536,limit_1) ([state](https://baike.baidu.com/item/%E8%A5%BF%E7%8F%AD%E7%89%99/148941)) | ([state](https://bigenc.ru/geography/text/2632850)) |

Wikipedia (flag page) represents the first image on the Wikipedia article corresponding to the flag.
Wikipedia (country page) represents the flag image on the Wikipedia article for the country itself.

(I've used only three countries, as examples, to try to determine whether this approach is at all worthwhile.)

**Comment by ukanuk** [comment]
If both flag versions are commonly used (including in Spain per @josealberto4444 above), I suggest showing the flag with the coat of arms with the footnote, "Flag without coat of arms is also common."

Implicitly teaching deck users there is just one flag version is a disservice when reality is more complex.

Moreover, even if the flag without coat of arms is used 80% of the time vs 20% of time with coat of arms, I think UG should show the version with the coat of arms. I want to be able to recognize both flag versions in the wild, and I think it's easier to recognize both if I just study the version with coat of arms.

**Comment by ollie299792458** [comment]
My two cents is:
 - 100% agree with @ukanuk suggestion of notes on flags, but I strongly disagree on @ukanuk 's second point
 - We should pick a rule and stick to it (going off of the Wikipedia Country page seems like a decent bet - outsourcing the debates to the talk pages)

**Comment by axelboc** [comment]
Mentioning the alternative civil/state flag below the flag makes sense. This might require adding a new field to the note template, but it wouldn't be a huge deal.

**Comment by KentGrigo** [comment]
I think that we should only have one source for information, otherwise we'll up making our own standard: https://xkcd.com/927/
It avoids making an intricate standard that is difficult to maintain.
It'll be cumbersome to check the consensus of all the sources for all the flags to check whether UG's out of date.
While the sources might agree on the civil/state flag for national flag, which source should we rely on for the colors and ratios? Because there's almost no agreement on that.

I just checked for Afghanistan which is the most recent country to change its flag (that I know of), and only Wikipedia is up-to-date.

Flag | Wikipedia (flag page) | Wikipedia (country page) | Britannica | Larousse | SNL | Baidu Baike | Great Russian Encyclopedia
-- | -- | -- | -- | -- | -- | -- | --
Afghanistan | [new](https://en.wikipedia.org/wiki/Flag_of_Afghanistan) | [new](https://en.wikipedia.org/wiki/Afghanistan) | [old](https://www.britannica.com/place/Afghanistan) | [old](https://www.larousse.fr/encyclopedie/pays/Afghanistan/104172) | [old](https://snl.no/Afghanistan) | N/A | [old](https://bigenc.ru/geography/text/4053221)

(Images didn't load for me on Baidu Baike)

Ideally, I'd like a scraper to fetch and update the flags, but that'll be significantly more difficult to do if we have multiple sources.

**Comment by aplaice** [comment]
Thanks everyone for the input!

I'll try to summarise our preferences for a standard:

1. Correct/reliable (obviously)
2. Stable in time

    Changing too frequently between two almost-equivalent options is worse, for learning, than sticking with one (even if it's the slightly worse one).

        - What's "too frequently"?
        
        - Minor stylistic changes (e.g. details in the coat of arms, precise colouring) are IMO more OK than major changes (e.g. choice between civil/state flag)
        
        - This obviously doesn't apply (or applies _much_ less) when there's an actual real-world change.

3. Simple

    One source is better than two etc.

4. Strong preference for (English) Wikipedia (for ideological, practical and conflict-avoidance reasons).

### Solutions

1. Use our current guidelines — flag from the country Wikipedia article — with lag.

    This is, in effect, what we do now.  The lag helps avoid too frequent changes.

    Unfortunately, given that the flags for several countries seem to continue oscillating between the options, with some (but not much) evidence for settling on one over the other, this still isn't great, even with lag.

2. Use the majority view from a combination of sources

    This could be some subset of (flag on Wikipedia country article, first flag on Wikipedia "flag of country" article, Britannica, Larousse, SNL, Baidu Baike, Great Russian Encyclopedia).
    
    As the number of sources increases, this becomes more-and-more unwieldy.
    
    If the majority is thin (4:3 or 2:1), then we can still end up with instability...
    
    What about ties?  (We can have an odd number of sources, but sometimes a source will have a clearly-wrong flag (e.g. not up-to-date), so it won't be included in the count.)
    
3. Default to the state flag in "case of doubt".

    Doubt could be where at least `n` out of our `m` sources (from 2.) disagree.
    
    Doubt could also be some sort of temporal averaging of Wikipedia.

I personally am conflicted between 1. and 3. (with sources=(flag on Wikipedia country article, first flag on Wikipedia "flag of country" article, Britannica) and one disagreement needed for doubt (m=3, n=1)), probably leaning towards 1.

<hr/>

### One, existing standard is usually better

> otherwise we'll up making our own standard: https://xkcd.com/927/

Yes, that's true. :D

> It avoids making an intricate standard that is difficult to maintain.

Yes, definitely, and if Wikipedia hadn't proven itself to be "unstable" in this regard, we wouldn't be considering anything like this.

> While the sources might agree on the civil/state flag for national flag, which source should we rely on for the colors and ratios? Because there's almost no agreement on that.
>
> I just checked for Afghanistan which is the most recent country to change its flag (that I know of), and only Wikipedia is up-to-date.

I think we can rely on Wikipedia to determine the precise designs of the flags and to be up-to-date on actual changes of the flag (as in the case of Afghanistan).  (As long as our system isn't fully automatised, it's (I think!) straightforward to distinguish between these (precise design and material changes) and choice of the "main" flag among civil/state/etc. flags, though, yes, it is cumbersome!)

However, I'm not sure that we can really rely on Wikipedia to determine the single, main flag.  Firstly, on empirical grounds, the flags of Venezuela, Bolivia, Peru and several other countries have "oscillated" over the years between the options (and continue to oscillate — possibly slower than previously?).


### "Side" suggestions

"Side" because I don't think that by themselves they solve the issue itself (what is the _main_ flag that we display).

1. Mention the alternative flags in our cards.

    This would likely require an extra field.

    We could even include images!
    
    If we decide that this is too heavy, we could also have an experimental super-extended "everything but the kitchen sink" variant deck.

    I really like the idea, but we should probably continue to discuss it in a separate thread.


### History of switching between flags on Wikipedia

I'll try to have a look at the histories and talk pages of the articles of the various countries with switching flags, to see how bad the problem still is, when I have time.

From a _very_ casual glance, Bolivia had the "civil" flag for several months in 2022 (but returned to the state flag), Costa Rica had the "state" flag for several months (but returned to the civil flag) and [Peru's talk page discusses the switching issue as a known problem](https://en.wikipedia.org/wiki/Talk:Peru#Peru's_National_Flag) (so at least for Peru, I think the choice of the civil flag as the "main" flag, is clear).

**Comment by axelboc** [comment]
Thanks for this great synthesis of the options at our disposal!

Personally, I'm leaning towards a combination of 1. and 3.:

- Sources: the country and flag pages on English Wikipedia.
- If both sources agree, we pick that; if they disagree, we default to the state flag.
- As usual, lag behind changes by a few months to reduce risk of back-and-forths.
- In case of repeated back-and-forths, investigate edit wars, make decision, and document decision in guidelines.

**Comment by ollie299792458** [comment]
I agree, but we should definitely add an explainer to countries where both are in common use.

**Comment by aplaice** [comment]
Case-by-case by analysis of edit wars for the flags affected by #587 (Venezuela, Peru, Costa Rica):

(For all three, both the country and flag articles currently have the civil flag as the "main" flag.)

## Peru

As noted previously in the thread, there seems to be consensus that the "correct" flag is the civil flag (so I think we should indeed switch):

https://en.wikipedia.org/wiki/Talk:Peru#Peru's_National_Flag

## Venezuela

The use of the civil flag in the article on Venezuela was for some time apparently because (see [here]((https://en.wikipedia.org/w/index.php?title=Venezuela&diff=prev&oldid=994423123)) and [here](https://en.wikipedia.org/w/index.php?title=Venezuela&diff=prev&oldid=1002320601)) the coat of arms in (the Wikimedia image of) the state flag [was wrong](https://commons.wikimedia.org/wiki/File_talk:Coat_of_arms_of_Venezuela.svg) (so they'd rather use the civil flag rather than a "wrong" state flag), but AFAICT the complaints in the discussion there have now been resolved (in both the coat of arms image and the derived state flag image).  However, our (AUG) flag of Venezuela still has the "old" coat of arms image ("old" as in Wikimedia history — AFAIU it's a hybrid of the actual pre-2006 and post-2006 designs), so we should either switch to the civil flag (like Wikipedia) or update the state flag (to the correct coat of arms design).  The former is probably saner.

(I _think_ it's OK to change to the civil flag.)

## Costa Rica

Costa Rica's article has used the state flag between 2016 and 2022 (with a very brief change to the civil flag in 2019-11 and a quick reversion). Between 2014 and 2016 and in 2022, there seems to have been a repeated back-and-forth between the civil and state flags. 

The main argument in favour of the state flag (in some of the edit summaries and talk pages) is that the state flag is the national flag; the main argument in favour of the civil flag is that the civil flag is the "bandera" (the main flag) and the state flag is only "Pabellón Nacional" (only used by government and similar bodies).

OTOH googling for `bandera de costa rica site:.cr` I mostly see pages with images of the state (not civil) flag.  OTOTOH for instance [this](https://www.mep.go.cr/noticias/dia-pabellon-nacional-0) describes the "pabellón nacional" as the flag + coat of arms.

On the whole, I'm personally not convinced either way, but given that the change on Wikipedia is recent and has had multiple reversions in the past year, I think I'd prefer AUG not to change our flag of Costa Rica, for now.

**Comment by axelboc** [comment]
Alright, seems legit 👍 

So if I summarize, we need to:

1. Update the contribution guidelines:
  - SVG only
  - state vs civil as summarized in https://github.com/anki-geo/ultimate-geography/issues/111#issuecomment-1357718585 
  - a list of the edit wars and corresponding decisions, with a link to @aplaice's https://github.com/anki-geo/ultimate-geography/issues/111#issuecomment-1382229132
  - point to @KentGrigo's https://github.com/anki-geo/ultimate-geography/pull/587#issuecomment-1357163000 on blurring SVG images instead of #247.
2. Replace/update/optimise the remaining flags in #587 as per the decisions above.

How do we go about adding the explainer when both civil/state flags are commonly used? Do we add a new field, reuse the flag similarity field, etc.?

**Comment by KentGrigo** [comment]
- I've removed the references to PNG for flags: https://github.com/anki-geo/ultimate-geography/pull/587/commits/f11387d650abebacfe8ed7e0c2f6ea40a5b343c2
- I've changed the reference to the description on blurring SVGs: https://github.com/anki-geo/ultimate-geography/pull/587/commits/9f90a7dffca01d0fb2a9d583e25de8e9d6c16691

**Comment by KentGrigo** [comment]
Just to be clear on what flags we want:
- Belize: `state` ✅ 
- Bolivia: `state` ✅ 
- Costa Rica: `state` (`civil` later?) ❌ (✅)
- El Salvador: `state` ✅ 
- Peru: `civil` ✅ 
- Venezuela: `civil` (preferred, otherwise update the `state` version) ✅ (❌)

Notation:
✅ means that https://github.com/anki-geo/ultimate-geography/pull/587 supports this.
❌ means that https://github.com/anki-geo/ultimate-geography/pull/587 doesn't support this.

So, I should only change the flag of Costa Rica back to being the **state** flag and then I can merge https://github.com/anki-geo/ultimate-geography/pull/587 ?

**Comment by aplaice** [comment]
> So if I summarize, we need to: [...]

Thanks for the summary @axelboc! (I fully agree! :))

> Just to be clear on what flags we want: [...]

Yes, exactly! And thanks for your patience!

> Costa Rica: ... (civil later?)

Quite possibly yes, (but as noted let's leave the state flag for now).

> How do we go about adding the explainer when both civil/state flags are commonly used?

Maybe let's open a new issue to track this (it's very closely related to this, but slightly separate and this thread is already pretty long)?

**Comment by horwitz** [comment]
@axelboc (et al.): Seeing as this has been open since 2019 and there's been no posted progress in almost a year, could we move this off of the v5.2 milestone (to some future (e.g., 5.3) milestone)?

**Comment by axelboc** [comment]
I think all that's left is to update the contributing guidelines with:

- how we decide between state vs civil as summarized in
https://github.com/anki-geo/ultimate-geography/issues/111#issuecomment-1357718585
- a list of the edit wars and corresponding decisions, with a link to @aplaice's
https://github.com/anki-geo/ultimate-geography/issues/111#issuecomment-1382229132

Should be doable, I reckon.

**Comment by axelboc** [comment]
Resolved with #627, thanks again @horwitz!
